lab_1/main.py

```diff
@@ -62,10 +112,23 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_to_compare, dict)
+            or not isinstance(top_n, int)):
+        return None
+    # use function get_top_n_words
+    top_n_words_unknown = get_top_n_words(unknown_profile["freq"], top_n)
+    top_n_words_compare = get_top_n_words(profile_to_compare["freq"], top_n)
+    # find common tokens
+    common_things = list(set(top_n_words_unknown) & set(top_n_words_compare))
```

Comment:
why list?

Reply:
I'm used to working with lists, so it seemed dangerous to me to just leave the set without creating a list, but now I understand that I can find the length of the set without problems, so the change is done